src/bd/primitives/Primitive.java

```diff
@@ -0,0 +1,84 @@
+package bd.primitives;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Repeatable;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+import com.oracle.truffle.api.dsl.NodeFactory;
+
+
+/**
+ * Annotation to be applied on node classes that are meant to be used as primitive operations.
+ * Thus, the node implements a built-in function of the interpreter, and might need to be
+ * represented as a language-level method/function.
```

Comment:
I know. What I mean is that the node in which the annotation is written is a ExpressionNode and the framework itself is in charge of creating an Invokable that includes that node. My point, and yes, it is just an opinion, is that someone not very involved with the framework, at a first read, may got confused and think that he should put the annotation in an Invokable like node.